src/ngraph/op/fused/gelu_grad.cpp

```diff
@@ -0,0 +1,75 @@
+//*****************************************************************************
```

Comment:
Do we need separate gelu_grad.cpp and gelu_grad.hpp? Some Gelu is already a fused op, gelu and gelu_grad can be merged together

Reply:
The coding standards actually say they shouldn't be combined, but I think that would be reasonable.

Reply:
I meant combining gelu_grad.cpp and gelu.cpp. And combining gelu_grad.hpp and gelu.hpp.